smart-connector/pom.xml

```diff
@@ -179,7 +179,7 @@
 		<swagger-core-version>1.5.18</swagger-core-version>

```

Comment:
Is there a reason why you did not change the `swagger-core-version` here to `1.6.12` as in the other locations?